content/components/data-table.mdx

```diff
@@ -544,7 +544,7 @@ If one of the actions is heavily used, pull it out for easier access. Do not pul
 
 ### Filtering
 
-A [filter input](/components/filter-input) is used to write a query and only show rows that
+A [filter input (internal)](https://ui.githubapp.com/storybook/?path=/docs/recipes-filter--readme) is used to write a query and only show rows that
```

Comment:
```suggestion
A [filter input (GitHub staff only)](https://ui.githubapp.com/storybook/?path=/docs/recipes-filter--readme) is used to write a query and only show rows that
```

I believe this is how we typically denote content in primer.style that is inaccessible to non-Hubbers.